src/ServicePulse.Host.Tests/SpecsRunner.html

```diff
@@ -28,6 +28,8 @@
     <script src="../servicepulse.host/app/lib/moment.min.js"></script>
     <script src="../servicepulse.host/app/lib/angular/angular-momentjs.js"></script>
     <script src="../servicepulse.host/app/lib/angular-ui-select/select.js"></script>
+    <script src="../servicepulse.host/app/lib/rx.angular/rx.all.js"></script>
```

Comment:
@WilliamBZA that seems to be a trivial thing which we could do ourselves which would bring us the benefit of referencing the rx libs directly (and have a better versioning)